ios/sdk/module/image loader/HippyImageLoaderModule.m

```diff
@@ -46,42 +47,72 @@ @implementation HippyImageLoaderModule
     }
     CFURLRef urlRef = CFURLCreateWithBytes(NULL, [uriData bytes], [uriData length], kCFStringEncodingUTF8, NULL);
     NSURL *source_url = CFBridgingRelease(urlRef);
-    __weak __typeof(self) weakSelf = self;
-	[[[NSURLSession sharedSession] dataTaskWithURL:source_url completionHandler:^(NSData * _Nullable data, __unused NSURLResponse * _Nullable response, NSError * _Nullable error) {
-        __typeof(weakSelf) strongSelf = weakSelf;
-		if (error) {
-            NSError *error = [NSError errorWithDomain:@"ImageLoaderModuleDomain" code:1 userInfo:@{@"reason": @"url parse error"}];
-            reject(@"2", @"url request error", error);
-		} else {
-            [[HippyImageCacheManager sharedInstance] setImageCacheData:data forURLString:urlString];
-            Class<HippyImageProviderProtocol> ipClass = imageProviderClassFromBridge(data, strongSelf.bridge);
-            id<HippyImageProviderProtocol> instance = [ipClass imageProviderInstanceForData:data];
-            UIImage *image = [instance image];
-			if (image) {
-				NSDictionary *dic = @{@"width": @(image.size.width), @"height": @(image.size.height)};
-				resolve(dic);
-			} else {
-                NSError *error = [NSError errorWithDomain:@"ImageLoaderModuleDomain" code:2 userInfo:@{@"reason": @"image parse error"}];
-                reject(@"2", @"image request error", error);
-			}
-		}
-	}] resume];
+    
+    typedef void (^HandleCompletedBlock)(BOOL, NSData *, NSURL *, NSError *);
```

Comment:
It's better to define HandleCompletedBlock in .h file, and keep  the same order of params.
[HippyImageViewCustomLoader.h line40](https://github.com/Tencent/Hippy/pull/449/files#diff-23f9fff918acb7636d0053c05e27b59b6a1a31dcd1f7a75ec5abc1a31af56630R40)

- -(void)loadImage:(NSURL *)url completed:(void (^)(NSData *, NSURL *, NSError *, BOOL cached))completedBlock;
- -void (^HandleCompletedBlock)(BOOL, NSData *, NSURL *, NSError *);